### DISCUSSION TITLE
Daily SCRUM

### DISCUSSION THREAD

**Discussion by katemalash** [IT-Academy-Social-Projects-KRV/CMIYC#16] [Show and tell]
We write our daily statuses here in format:

Y: What I've done yesterday
T: What I'll do today
B: What blockers do I have

**Comment by katemalash** [comment]
15 Feb 22

**Reply by PavelHorishny** [reply]
> 15 Feb 22

Y:I added template to test angular project
T:I will try to make some components using template

**Reply by RomanTertychnyi** [reply]
created a spring boot app and try how to work in it.
I want to try to connect to the database

**Reply by MrGold6** [reply]
Y: Created angular project, clear html, add router
T: Add auth guard, finish git ignore, create component
B:  Where I can find information about git ignore (what should I add)? Should we really use auth guard?

**Reply by TetyanaYahodska** [reply]
> 15 Feb 22
Y: I studied how components interact in Angular, how data services work.
T: I will complete my task when I practice a little more in WebStorm. 
B: Difficulties arose only with understanding a large quantity of information.

**Reply by Ajderka** [reply]
Y: I was sorting out the tasks, making sense of them. 
T: Created a REST project and figured out the technologies.
B: Finish a REST Spring application using the necessary technologies.

**Reply by Gorbokonyk** [reply]
Y:  I created Logo and favicon, created some components(footer, header).
T: Create other Angular components for given project.
B: -

**Reply by yarikfragik** [reply]
Y: Watched some documentation about angular and MS sql
T:Setup database , create simple app with angular

**Reply by YaroslavMalyarskyy** [reply]
Y: I was searching templates for project.
T.:I found two angular templates and several other templates.
B: I don't know which template we need.

**Reply by TetyanaYahodska** [reply]
Y:  I worked with Git. I understood Bootstrap, was able to connect and try some elements of a test project in a web storm. 
T: I want to make a component and display the desired shape on my page.
B: I solved all the problems with the help of the guys.

**Comment by Ajderka** [comment]
16 Feb 22

**Reply by Camper689** [reply]
Y: read documentation to spring authorization server
T: will try to create simple application with oauth2 consent screen.

**Reply by Ajderka** [reply]
Y: Made requests to REST server, received object in JSON format. Figured out some of the technology. 
T: Read Spring Microservises in action book, followed the steps in the book, installed Docker, configured the system to make it run.
B: Next steps in the book to learn more about the topic. In particular with the technology stack used in Micriservises. 
At this stage, I have no blocks.

**Reply by MrGold6** [reply]
Y:  finished git ignore, created pull request
T: Create component, try to use template
B: We can't understand what components we should do, without design for component

**Reply by TetyanaYahodska** [reply]
Y: I did a test project with Reactive Forms with a set of FormGroup and FormControl objects. Checked how to prescribe different field validation. 
T: I want to finish the work started with the settings of our project.
B: I had problems with the settings of the dependencies of our skeleton

**Reply by PavelHorishny** [reply]
> 16 Feb 22

Y:I was adding templates to existing test porjects and making some forms using templates.
T: I'll find custom error pages

**Reply by Gorbokonyk** [reply]
Y: Continued practicing with Angular, routing, creating modules, auth.
T: Understanding of model structure of project.
B: Dependencies, what model we should use.

**Reply by YaroslavMalyarskyy** [reply]
Y: I tried to add dependencies from template to project.
T: I want to add register page.
B: I have dependencies conflicts.

**Reply by ZX-pool** [reply]
Y: I worked with simple REST application
T: I will think about how to made skeleton of application
B: We have not decided yet about version of Java we will be use

**Reply by JohnnyMahoney** [reply]
Y:Watched some new videos about auth.
T:Thinking about creating easy auth application

**Reply by RomanTertychnyi** [reply]
Y: Created an easy REST application
T: Download Postman and work with queries

**Comment by Ajderka** [comment]
17 Feb 22

**Reply by Camper689** [reply]
y: created simple spring boot authorization server, installed docker
t: will read documentation to docker

**Reply by PavelHorishny** [reply]
> 17 Feb 22

Y: I found error pages templates and added them to test project
T: I'm going to put them to repo and find out how to connect them to angular correctly

**Reply by ZX-pool** [reply]
Y: I have done simple application with database and REST controller
T: work on skeletons of microservices
B: Usually we use mapping on database with some entity. What kind of mapping we will use in our Search API or Connector API

**Reply by MrGold6** [reply]
Y: i've done design for admin pages
T: create components for admin pages  
B: we should understand what design product owner want

**Reply by Ajderka** [reply]
Y: Installed a Java container on Docker. I was figuring out the moments of conversion to Json format with Jackson. Installed POSTMAN. Made some server requests...
T: Took theory material about REST APIs, SPRING ORMs, and the concept of OAUTH2.
B: Finish with SRING BOOT theory and write first application with a skeleton.
Blocks : No understanding of how to work with Docker in the context of a JAVA application.

**Reply by Gorbokonyk** [reply]
Y: Connected template to UI.
T: Create search page for user, create modules for angular project.
B: Understanding how frontend work/connect with specific microservices.

**Reply by YaroslavMalyarskyy** [reply]
Y: I was writing HTML code for the project, but did not finish 
T: I plan to finish it.
B: I didn't have any blokers.

**Reply by RomanTertychnyi** [reply]
Y: worked with REST and executed queries in Postman.
T: I want to understand  more Git.

**Reply by JohnnyMahoney** [reply]
Y:Made a simple spring project,watched how to working with a postman,watched videos about auth
T:continue working with a spring project,continue working with auth service
B:some blockers with a java syntax,want to look back at some moments.

**Comment by Ajderka** [comment]
18 Feb 22

**Reply by YaroslavMalyarskyy** [reply]
Y: I created sidebar component.
T: I'm going to create header component
B: I don't have any bloks.

**Reply by Ajderka** [reply]
Y: Finished learning Spring Boot theory, got interested in Spring Data REST.
T: Practiced in building an application in Spring Data REST.
B: Create a working service to demonstrate Spring Data REST functionality to the team.

**Reply by ZX-pool** [reply]
Y: read some tutorials about OAuth technology and how it works
T: I`m going to prepare my equipment to work: reinstall windows 10 and all needed programs 
W (weekend): try to create simple REST application or skeleton of one of microservices
B: it is needed more information about our project

**Reply by TetyanaYahodska** [reply]
Y: I created a login form component, researched how to connect two-factor authentication to the angular project.
T: I want to try to do two-factor authentication on a test project so as not to break anything in our project.
B: How to set up a git so that it works like everyone else.

**Reply by MrGold6** [reply]
Y: Created components for admin
T: Push my update
B: Waiting to merge sidebar with my project

**Reply by JohnnyMahoney** [reply]
Y:worked with auth server information
T:today or tomorrow we will finish our task for auth skeleton
B:nothing

**Reply by Gorbokonyk** [reply]
Y: Edit user page, create angular component user page.
T: Continue to edit user page.
B: No blocks.

**Reply by RomanTertychnyi** [reply]
Y: I read the documentation for spring boot.
T: Download and install the docker and read the documentation.
B: don't have bloks

**Comment by Ajderka** [comment]
21 Feb 22

**Reply by Ajderka** [reply]
**Y** : Made a skeleton of microservices, ran them all 3 at the same time. Also practiced with git together with the team. Started watching the docker course.
**T** : Continued watching Docker course. I plan to build images and create images of our microservices.
**B** : I don't have any blocks.

**Reply by MrGold6** [reply]
Y : I've made components for admin, merge it with side bar component, add routers, push and merge
T :  I'm going to install Docker and prepare to demo
B :  I don't have any blocks

**Reply by TetyanaYahodska** [reply]
Y: Researched how to make two-factor authentication, worked with the preparation of our project for the demo.
T: I want to get information in preparation for two-factor authentication.
B : I don't have any blocks

**Reply by ZX-pool** [reply]
Y: I prepared equipment to fully work. Reinstalled windows 10 and installed all needed programs. Worked in team with git and microservices
T: I'm going to prepare to demo
B: No

**Reply by Camper689** [reply]
Y: Created empty auth server, added mysql database support and flyway migrations. Added migration that create tables in database. Added simple README. T: will continue to read documentation to spring authorization server

**Reply by YaroslavMalyarskyy** [reply]
Y: Researched how to make form validation in angular project
T: I want to add validation for my test project.
B : I don't have any blocks

**Reply by PavelHorishny** [reply]
> 21 Feb 22

Y: I've done my tasks and pushed it to repo.
T: i'm going to start reading book about Spring.
B: I met some troubles with adding changes to my pull request, but my teammates helped me to solve it.

**Comment by Ajderka** [comment]
22 Feb 22 - DEMO

- Let me give you an overview of our CMIYC project. The idea of the application is to provide information to a police officer according to his request about a potential intruder. The application includes functions of authentication, authorization, conversion of data from several sources according to the template. The visual part with the functional possibilities to select the information source and provide the information in a user-friendly form was made as well. During the development the security of access to the data was accepted as the priority.

- So, today the developers will present you the project, which has been partially implemented according to the set tasks. The purpose of our meeting is basically to check the clock and determine the direction of further development.

**Reply by Ajderka** [reply]
Yesterday: a team meeting was held in which roles were assigned to all participants and critical tasks were discussed. A common plan of action was adopted.

Today: Preparation for DEMO.

Blocks: No blocks

**Reply by PavelHorishny** [reply]
Y: I started read about Spring.
T: I'm going to initialize test Spring project.

**Comment by Ajderka** [comment]
23 Feb 22

**Reply by Ajderka** [reply]
Y: practiced making authentication applications from the beginning to understand how this microservice works. 
P: finish the authentication tutorial project and start problem solving.
B: No blocks
- I won't be able to actively participate in our meeting today, just listening...

**Reply by PavelHorishny** [reply]
> 23 Feb 22

Y:I initialized SpringBoot test project, figured out how to use SpringInitializr.
T: I'll continue reading Spring in action and practice.

**Reply by MrGold6** [reply]
Y: reading about microservices
P: Will start my tasks, what I will get
B: No blocks
M: I will be a bit late on meeting

**Reply by Gorbokonyk** [reply]
Y: Spring practice(auth).
T:  Spring practice
B: No blocks

**Reply by JohnnyMahoney** [reply]
T:going to learn microservices as it my task for the next spring

**Comment by ZX-pool** [comment]
17 Mar 22

Y: work on demo app using RestTemplate and Feign client
B: some trouble with OpenFeign dependencies
T: continue work on demo app

**Reply by MrGold6** [reply]
Y: i tried to setup project Auth + Angular + Spring
B: I has some bugs with this project
T: I will try to setup it or change demo project, also I will create demo API project with swagger to understand how it works

**Reply by Gorbokonyk** [reply]
Y: Spring practice, build Auth service.
B: Understanding Spring security.
T: Setup demo Spring REST API OAuth2 service.

**Reply by YaroslavMalyarskyy** [reply]
Y: I was reading the angular documentation about forms validation.
T: I tried to add a validation for the registration component.
B: My knowledge of TypeScript.

**Reply by JohnnyMahoney** [reply]
Y:repeat some annotations,new topics and videos about microservises
T:want to play with Spring Boot and will be trying to make some connections with microsevices
B:knowledge of Spring

**Reply by PavelHorishny** [reply]
> 17 Mar 22
Y:Start looking for info about cloud services
T:Will continue research and make short report

**Comment by ZX-pool** [comment]
18 Mar 22

**Reply by ZX-pool** [reply]
Y: work on demo app. Connect 2 services using WebClient
T: continue work on demo app
B: no blocks

**Reply by YaroslavMalyarskyy** [reply]
Y: I created a test project with a registration form and added a validation to this form.
T: I'm going to add a validation to the registration component.
B: I have no blocks.

**Reply by PavelHorishny** [reply]
Y: I finished research and made brief presentation for found information
T: I'm going to continue learning Spring
B:No blocks

**Reply by MrGold6** [reply]
Y: work on demo Rest Api app, connected it with swagger, created simple crud
T: continue work on demo Rest Api app
B: no blocks

**Reply by Gorbokonyk** [reply]
Y: Learn Spring
T: Continue learning Spring
B: No blocks

**Reply by JohnnyMahoney** [reply]
Y:worked with information about microservices,learn some information about JWT,continue learning Spring
T:we are going to decide which connection we use for our microservices,so continue working with it
B:lack of practice of coding

**Comment by ZX-pool** [comment]
21 Mar 22

**Reply by MrGold6** [reply]
> 21 Mar 22

Y: I've finished rest api demo app
T: I will start to create prototype of Search API on paper (on Wednesday will start coding)
B: no blocks

**Reply by TetyanaYahodska** [reply]
Y: I studied the materials about Authguard, launched a project on a new laptop.
T: I want solve my problems with the settings and want to make a prototype Authguard.
B:  I tried to make a prototype of Authguard for our project, but after several changes and attempts in the versions of Angular there were errors that are still unresolved.

**Reply by YaroslavMalyarskyy** [reply]
Y: I added a validation for registration, login and two-factor authorization.
T: I'm going to add validation to the search component
B: I have no blocks.

**Reply by JohnnyMahoney** [reply]
Y:watched videos and information about feign client,started learning AOP to best skills
T:continue looking for new information,maybe try to connect microservices
B:difficulties of how do what i watched on our project

**Reply by ZX-pool** [reply]
Y: I finished my demo app. Eureka server added
T: I will learn about Auth2 protocol and howto implement Resource server in microservices
B: No block

**Reply by PavelHorishny** [reply]
> 21 Mar 22

Y:I learnt about SOAP requests, WSDL and XML. Build Spring SOAP API using guides.
T:I'm going to learn how to send SOAP requests and receives responses.
B:No blocks.

**Comment by ZX-pool** [comment]
22 Mar 22

**Reply by ZX-pool** [reply]
Y: read about JWT tokens
T: I will start work on Conector API
B: no blocks

**Reply by PavelHorishny** [reply]
> 22 Mar 22

Y:I finished my SOAP app, successfully sent SOAP request and received response using command line. Started working on WebSocket echo server.
T: I want finish WebSocket server.Also I want to learn how to send SOAP request from app.
B: No blocks.

**Reply by YaroslavMalyarskyy** [reply]
Y: I finished the validation for registration, login, and two-factor authorization and added some validation for the search component, but I didn't complete it.
T: I'm going to finish validation for the search component today.
B: I have no blocks.

**Reply by Ajderka** [reply]
Y: found the right material, at this stage in the process of solving the problem
T: going to finish email issue
B: I have no blocks

**Reply by Gorbokonyk** [reply]
Y: Working on connection UI with backend services using.
T: Going to finish my task today.
B: I have no blocks.

**Reply by JohnnyMahoney** [reply]
Y:continue working with microservices especially feign client,continue learning AOP 
T:continue working with it
B:no blocks

**Comment by ZX-pool** [comment]
23 Mar 22

**Reply by MrGold6** [reply]
Y: drew architecture of Search API on paper
T: setup project, try to connect with auth
B: for architecture : will  Search API convert data due to schema?

**Reply by YaroslavMalyarskyy** [reply]
Y: I worked with validation for the search component.
T: I'm going to continue the same.
B: Problems with HTML layout.

**Reply by ZX-pool** [reply]
Y: i worked with Controller API
T: continue working with Controller API and try realize Resource Server
B: no blocks

**Comment by ZX-pool** [comment]
24 Mar 22

**Reply by ZX-pool** [reply]
Y: worked with Controller API and Resourse server
T: continue working on Resourse server
B: no block

**Reply by TetyanaYahodska** [reply]
Y: I downloaded MySQL and worked on the AuthGuard prototype in our project.
T: I will continue to study Spring Security and AuthGuard,  and work on a AuthGuard prototype.
B : no block

**Reply by YaroslavMalyarskyy** [reply]
Y: I added validation for search component.
T: I'm going to create reusable form component.
B: No Blocks.

**Comment by ZX-pool** [comment]
25 mar 22

**Reply by MrGold6** [reply]
Y: I've connected Data API and Connector API with Search API, write documentation how to do it
T: Finish basic architecture of Search Api, mb connect with Auth, add swagger
B :  waiting for tutorial of resource server, can't setup auth service with database.
Have a questions of realization:
1. where should  I safe result from connector/data/ui? is it object from model? 
2. should i set to connector or data what api i need?
3. at front i should get names of api before i will search or when?

**Reply by ZX-pool** [reply]
Y: worked with Connector API and Resourse Server
T: create Pull request
B: no block

**Reply by YaroslavMalyarskyy** [reply]
Y: I have completed my task and created the pull request. Also yesterday I started working on my second task.
T: I'm going to continue the same.
B: No blocks.

**Comment by ZX-pool** [comment]
28 mar 22

**Reply by MrGold6** [reply]
Y: I created Search API prototype, connect it with Connector, Data, Auth, created class diagram (link in Search api documentation)
T: I'm going to prepare for demo
B: I tried to connect Swagger, but I can't understand how to do response example, so I will do it for the next sprint

**Reply by ZX-pool** [reply]
Y: finished work with Connector API
T. preparation to demo
B: no

**Reply by PavelHorishny** [reply]
> 28 mar 22

Y: I've made two working server side apps (SOAP and WebSocket), also two client side apps, that can connect to server, send requests and receive responses.
T: I'm going to resolve issue that concern building WebSocket app using Spring.
B:No blocks.

**Reply by YaroslavMalyarskyy** [reply]
Y: I finished my tasks.
T. I'm going to preparation for the demo
B: No Blocks

**Comment by JohnnyMahoney** [comment]
### 1 april 22

**Reply by JohnnyMahoney** [reply]
Y:read a documentation about Angular,write a simple project
T:continue read,on weekends going to take a task and do it
B:knowledge of Frontend

**Reply by Camper689** [reply]
Y: Read documentation to Angular, take two tasks
T: Will work on tasks
B: No blocks

**Reply by PavelHorishny** [reply]
Y: Collected useful links about WebSocket
T: Will collect useful links about SOAP. Going to close investigation tasks.
B: Correct order of actions with scrum board in case if you have investigation task.

**Reply by YaroslavMalyarskyy** [reply]
Y: I was researching how to add two-factor authentication to our project.
T: I'm going to do the same.
B: No blocks

**Reply by ZX-pool** [reply]
Y: read about GCP
T: continue read about GCP, Docker, Kubernetes and other features
B: no

**Reply by Gorbokonyk** [reply]
Y: Read Docker Documentation.
T: Will create Docker container, and run it.
B: Nothing.

**Comment by YaroslavMalyarskyy** [comment]
4 april 22

**Reply by YaroslavMalyarskyy** [reply]
Y: I'm almost done creating prototype of  two-factor authentication.
T: I'm going to finish this task
B: My knowledge of qr-code

**Reply by Camper689** [reply]
Y: finished my task, created url mapping convention guide on wiki
T: started working on another task, will try to finish it today
B: no blocks

**Reply by MrGold6** [reply]
Y: Start investigating testing 
T: Continue investigating
B: No blocks

**Reply by JohnnyMahoney** [reply]
Y:was working on my task
T:try to finish it
B:question in UI

**Reply by ZX-pool** [reply]
Y: worked with docker. Built first docker container and run 
T: read about  kubernetes
B: no

**Reply by PavelHorishny** [reply]
Y: Collected useful materials for investigations.Learned git. Added links to issues. 
T: Will make skeleton for APIs.
B: No Blocks

**Comment by ZX-pool** [comment]
5 april 22

**Reply by ZX-pool** [reply]
Y: worked with docker and find elegant solution how to connect services inside docker
T: will start use "kind kubernetes"
B: no

**Reply by JohnnyMahoney** [reply]
Y:finished my task
T:going to do one more task
B:no blocks

**Reply by Camper689** [reply]
Y; finished my task
T: will start reading documentation to kubernetes, docker etc
B: -

**Reply by PavelHorishny** [reply]
Y:I made skeleton for external apis.
T:Going to do pull request and take next task
B: Not blocks rather some questions to ask and idea to discuss.

**Reply by Gorbokonyk** [reply]
Y: Tried to deploy our Authority microservice on docker container, work with docker, learn Kubernetes.
T: Try to deploy docker container to  Kubernetes.
B: error while running our Authority microservice .jar file.

**Reply by TetyanaYahodska** [reply]
Y: I studied how to download and update files from the frontend. 
T:I would like to do my task.
B: I need a more detailed explanation of my task

**Comment by JohnnyMahoney** [comment]
### 6 April 22

**Reply by JohnnyMahoney** [reply]
Y:read new information about servlet,tomcat
T:continue learning,try to finish my task
B:-

**Reply by YaroslavMalyarskyy** [reply]
Y: I finished my first task and start work on my second task.
T: I'm going to continue working on my second task
B: No blocks.

**Reply by PavelHorishny** [reply]
> ### 6 April 22
Y: Made two variants of Websocket server using different libs: javax.websocket.* and org.java_websocket.*. Also worked with java Servlets.
T: When we choose which technology we will use for our external services I'm going to make changes to skeleton.
B: I have some questions about REST architecture.

**Reply by ZX-pool** [reply]
Y: woked with kind kubernetis
T: continue work 
B: no

**Reply by TetyanaYahodska** [reply]
Y :I was working on a Data Api connection with an admin user.
T: I want to show the team to know what changes need to be made and already close this task.
B: Thanks to Yaroslav for helping with the blocks.

**Reply by Camper689** [reply]
Y: started reading documentation on docker, ran auth service in container
T: will continue reading documentation and maybe will start working on another task
B: -

**Comment by JohnnyMahoney** [comment]
### 7 april 22

**Reply by JohnnyMahoney** [reply]
Y:finished my task,prepared to the demo
T:showing my work on demo, trying to do the task
B:-

**Reply by PavelHorishny** [reply]
> ### 7 april 22
Y: Made pull request. prepared for demo
T: Will work on next task
B: No blocks

**Comment by JohnnyMahoney** [comment]
### 8 April 22

**Reply by JohnnyMahoney** [reply]
Y:we had a demo
T:some investigation work
B:-

**Reply by TetyanaYahodska** [reply]
Y: I investigated  how to upload our files to the bucket.
T: I will continue to study this topic and at the same time do it on my branch.
B: Problems with setting the dependency and creating beans

**Reply by Camper689** [reply]
Y: fixed two bugs
T: will start working on task
B: -

**Reply by ZX-pool** [reply]
Y: created GCP account
T: learn and work with GCP
B: no

**Reply by PavelHorishny** [reply]
> ### 8 April 22

Y:Worked with JavaEE, tried to handle external api connection with data source, worked with JSON.
T:will work on external rest api.
B: No blocks

**Comment by JohnnyMahoney** [comment]
### 11 april 22

**Reply by JohnnyMahoney** [reply]
Y:finished my task
T:preparing for demo
B:-

**Reply by ZX-pool** [reply]
Y: worked with GCP and Kubernetes. Start MySQL database in kubernetes using Persistent Volume and Secret key
T: will try connect microservices together in kubernetes
B: no

**Reply by PavelHorishny** [reply]
> ### 11 april 22

Y: Finished my tasks
T: When approves be ready will merge requests
B:No blocks

**Comment by Gorbokonyk** [comment]
**15 April 2022**

**Reply by PavelHorishny** [reply]
> **15 April 2022**

Y: Found out several different ways to add validation
T: Will check which approach was used before and try to implement it
B: Do we need file type validation?

**Reply by ZX-pool** [reply]
Y: finished work on deployment demo application to kubernetes in gcloud
T. start read about Ci CD technology
B: no

**Reply by JohnnyMahoney** [reply]
Y:read information about Docker and Kubernetes
T:continue working
B:no

**Reply by Gorbokonyk** [reply]
Y: Change access_token expiration time
T: Refactor email service
B: -

**Reply by MrGold6** [reply]
Y: Start investigate testing
T: Finish investigating and do demo app with testing
B: No blocks

**Comment by Gorbokonyk** [comment]
**18 April 2022**

**Reply by PavelHorishny** [reply]
> **18 April 2022**

Y:Added file size validation on admin schema page
T:Will try close this task and take next one.
B: What size limit should we use and what to do in case of sending empty field

**Reply by ZX-pool** [reply]
Y: read about CI/CD technologies. try write simple hello world CI/CD pipeline using gcloud
T: continue work with CI/CD
B: no

**Reply by JohnnyMahoney** [reply]
Y:read about docker and cubernetes,took a view on CI/CD
T:continue working
B:what should be in CD/CI

**Reply by YaroslavMalyarskyy** [reply]
Y: I tried to add two-factor authentication, but it didn't work.
T: I'm going to add two-factor authentication in another way.
B: I don't understand how to add two-factor authentication without overwriting the authorization server.

**Reply by Camper689** [reply]
Y: Investigated how to use H2 database and how to send HTTP request in java. Created two demo projects.
T: Will continue work on task
B: have some questions about search API.

**Reply by MrGold6** [reply]
Y: Investigated tests
T: How result of investigating for team, create some tests
B: No blocks

**Comment by Gorbokonyk** [comment]
**19 April 2022**

**Reply by PavelHorishny** [reply]
> **18 April 2022**

Y: work on file size validation
T:will continue working on task

**Reply by YaroslavMalyarskyy** [reply]
Y: I investigated how to configure spring security for two-factor authentication.
T: I'm going to add two-factor authentication using investigate results.
B: No blocks.

**Reply by ZX-pool** [reply]
Y: learn how to work CI CD automatic deployment to Cloud Run
T: i will try create simple pipeline for automatic deployment to GKE
B: no

**Reply by Camper689** [reply]
Y: finished my task, started working on the next task
T: going to finish my task and create pull request
B: -

**Reply by Gorbokonyk** [reply]
Y: finished my task
T: work on another task, create pull request
B: -

**Comment by Gorbokonyk** [comment]
**20 April 2022**

**Reply by Camper689** [reply]
Y: Added connection between connector API and external-apis
T: Will try to create some tests
B: -

**Reply by YaroslavMalyarskyy** [reply]
Y: I added a temporary access token for two-factor authentication
T: I'm going to send a QR code to the user's email
B: No blocks

**Reply by Gorbokonyk** [reply]
Y: Work on task update delete operation
T: Finish my task
B: -

**Reply by PavelHorishny** [reply]
> **20 April 2022**

Y:Worked still on validation
T: Will try to find good solution

**Reply by MrGold6** [reply]
Y: Created unit test manual (without Mockito) and test project.
T: Tell team about unit testing, add to manual Mockito and integration testing, create test project with integration testing, add unit tests to our project
B: I can't write unit test for our project due to not understanding how to get access token, can't do code coverage because mvn not working.

**Reply by TetyanaYahodska** [reply]
Y: I continued to work on solving my problems. But now I can't run the program and I don't know how to solve it.
T: I will work on solving the problem
B: Spring does not create a storage bean.

**Comment by MrGold6** [comment]
09 May 22

**Reply by MrGold6** [reply]
Y: Done pagination and added scroll for user responses
T: Create partial search for admin user
B: Should understand that scroll is ok for our product

**Reply by YaroslavMalyarskyy** [reply]
Y: I finished my first task, fixed the Cors Filter error, and created pull requests.
T: I'm going to create a dockerfile for external APIs
B: No blocks.

**Reply by ZX-pool** [reply]
Y: worked with dockerize and deploy frontend
T: continue work with frontend
B: frontend do not see other microservices in cloud cluster

**Reply by PavelHorishny** [reply]
Y: added accordion and pagination on search response page
T: continue work on tasks
B: no blocks

**Comment by MrGold6** [comment]
11 May 22

**Reply by MrGold6** [reply]
Y: Merged pull requests, continue work with search
T: Finish work with partial search
B: no blocks

**Reply by PavelHorishny** [reply]
> 11 May 22

Y:added pagination on admin user page and search response page using angular pagination module. Fixed links on user page
T:Modify pagination to work with backend pagination.
B: no blocks

**Reply by YaroslavMalyarskyy** [reply]
Y: I created dockerfiles for external APIs
T: I'm going to finish this task.
B: I can't connect to websocket API with postman

**Comment by MrGold6** [comment]
12 May 22

**Reply by MrGold6** [reply]
Y: Created search
T: Try to improve search, finish with this task
B: we need search on roles?

**Reply by PavelHorishny** [reply]
> 12 May 22

Y: Added accordion and pagination to search result page
T: Going to add pagination on admin user page
B: I can't make first accordion item to be opened. I have questions about backend pagination. Can I close task #126

**Reply by YaroslavMalyarskyy** [reply]
Y: I finished the task of dockerize external APIs and started the task of dockerize microservices
T: I'm going to continue working on my task
B: no blocks.

**Reply by ZX-pool** [reply]
Y: created public URL for authority server and test work frontend with authority server
T: will learn howto get external envirement variables inside angular and wait other dockerfiles in main branch to continue deployment
B: no

**Comment by MrGold6** [comment]
13 May 22

**Reply by MrGold6** [reply]
Y: Created search, add pull request
T: Will start write tests for email service
B: no blocks

**Reply by PavelHorishny** [reply]
> 13 May 22

Y:Worked on admin user pagination
T:will fix bug that concerns showing results
B:no blocks

**Reply by ZX-pool** [reply]
Y: i deployed External APIs to cloud
T: continue deploy other microservices
B: no

**Reply by YaroslavMalyarskyy** [reply]
Y: I added runtime variables for the user interface and created Docker files for microservices.
T: I'm going to continue work on my task.
B: In the docker, the command "mvn clean package" does not work.

**Comment by MrGold6** [comment]
16 May 22

**Reply by JohnnyMahoney** [reply]
Y:finished task,made a merge
T:gonna solve another task
B:no

**Reply by MrGold6** [reply]
Y: Tried to do my my task
T: Will start do another task, ui for search or tests for user service
B:  don't understand what should I test at email service

**Reply by YaroslavMalyarskyy** [reply]
Y: I started work on task "add loading animation" 
T: I'm going to continue work on this task
B: No blocks.

**Reply by ZX-pool** [reply]
Y: started deploy all services to cloud
T: continue deploying
B: services not connecting in cloud

**Reply by PavelHorishny** [reply]
> 16 May 22

Y:Made some changes on admin user page
T:Going to take next task
B:no blocks

**Reply by Ajderka** [reply]
Y: I made a pull request to change the password specified in the yml file to envoriment variables.
T: I'm currently working on a query history task, cache history
B: How to resonate stories?

**Comment by MrGold6** [comment]
17 May 22

**Reply by MrGold6** [reply]
Y: Finishing task with email service test
T: Will start do return page for search result
B: need we test template in main directory?

**Reply by Gorbokonyk** [reply]
Y: Finishing task with auth.dockerfile and email sending on registration
T: Will start new task edit delete user on ui
B: -

**Reply by PavelHorishny** [reply]
> 17 May 22

Y:I added delete user operation on UI
T:Going to finish work on update user operation on UI
B:No blocks

**Reply by YaroslavMalyarskyy** [reply]
Y: I started working on task "Create accordion view for response"
T: I'm going to continue working on this task
B: no blocks

**Reply by Ajderka** [reply]
Y: changed user token check to read request history
T: I plan to finish and going to change the return types 
B: no blocks
